WORKSPACE

```diff
@@ -0,0 +1,124 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+### Bazel rules for many languages to compile PROTO into gRPC libraries
+http_archive(
+    name = "rules_proto_grpc",
+    sha256 = "bbe4db93499f5c9414926e46f9e35016999a4e9f6e3522482d3760dc61011070",
+    strip_prefix = "rules_proto_grpc-4.2.0",
+    urls = ["https://github.com/rules-proto-grpc/rules_proto_grpc/archive/4.2.0.tar.gz"],
+)
+
+load(
+    "@rules_proto_grpc//:repositories.bzl",
+    "bazel_gazelle",
+    "io_bazel_rules_go",
+    "rules_proto_grpc_repos",
+    "rules_proto_grpc_toolchains",
+)
+
+rules_proto_grpc_toolchains()
+
+rules_proto_grpc_repos()
+
+load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_proto_toolchains")
+
+rules_proto_dependencies()
+
+rules_proto_toolchains()
+
+### Golang
+io_bazel_rules_go()
+
+load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
+
+go_rules_dependencies()
+
+go_register_toolchains(go_version = "1.18")
+
+# gazelle:repo bazel_gazelle
+bazel_gazelle()
+
+load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
+
+go_repository(
```

Comment:
Is this needed? It does not seem to be referenced in any of the BUILD files.

Reply:
nope fixed